frontend/src/views/main/project/InspectorWrapper.vue

```diff
@@ -296,7 +301,7 @@ async function submitValueVariationFeedback(userData: object) {
 
 const debouncedUpdateRow = _.debounce(async () => {
   await updateRow(false);
-}, 150);
+}, 250);
```

Comment:
Maybe could add `leading` option ?

```js
}, 250, { leading: true});
```

Reply:
The issue is that for long running models we want to avoid hitting them at all if the user starts fast navigation